src/components/_modal/pages/order/Order.page.js

```diff
@@ -92,7 +92,7 @@ function Order({
 
   function updateModal() {
     if (modal && modal.isVisible) {
-      // call update if data or isLoading is changed
+      // call update if data or isLoading or error has changed
```

Comment:
Skal `articleOrderMutation.error` og `orderMutation.error` også være nede i `useEffect`'en (her linje 113), evt? Kan ellers måske være et problem hvis den giver en error

Reply:
det overvejede jeg også, error'en kommer dog frem, jeg tænker det er fordi status på "isLoading" skifter også når vi får en error... måske har man ikke en gang brug for .data. 
Prøver måske lige at tilføje .error og kun med .isLoading...

Reply:
Nåh ja, isLoading vil sikkert være nok! God pointe

Reply:
det virkede fint med isLoading... så lad os håb testene er god nok og vil fange problemer :-)